optimum/onnxruntime/configuration.py

```diff
@@ -43,3 +47,6 @@ def __init__(self, **kwargs):
         self.split = kwargs.pop("split", "train")
         self.max_samples = kwargs.pop("max_samples", 80)
         self.calib_batch_size = kwargs.pop("calib_batch_size", 8)
+        self.nodes_to_quantize = (kwargs.pop("nodes_to_quantize", []),)
+        self.nodes_to_exclude = (kwargs.pop("nodes_to_exclude", []),)
```

Comment:
Could we keep `self.nodes_to_quantize` and `self.nodes_to_exclude` as lists ?

Reply:
Thanks for catching, it's a mistake, sorry about that 🙏🏻